src/perl/lib/wtsi_clarity/epp/generic/stamper.pm

```diff
@@ -8,6 +8,7 @@ use URI::Escape;
 use XML::LibXML;
 use Mojo::Collection 'c';
 use Try::Tiny;
+use List::MoreUtils qw(uniq);
```

Comment:
I'm not sure we are really using 'namespace::autoclean' module.
Could you check it, please?